eva_assembly_ingestion/assembly_ingestion_job.py

```diff
@@ -173,10 +196,12 @@ def process_one_assembly(self, source_assembly, instance, resume):
         remap_cluster_config_file = os.path.join(assembly_directory, 'remap_cluster_config.yaml')
         remapping_required = self.check_remapping_required(source_assembly)
         remap_cluster_config = {
-            'taxonomy_id': self.taxonomy,
+            'taxonomy_list': taxonomy_list,
             'source_assembly_accession': source_assembly,
             'target_assembly_accession': self.target_assembly,
-            'species_name': self.scientific_name,
+            # the actual species name does not need to match the taxonomy
+            # since it is only here to locate the fasta/report files
+            'species_name': self.scientific_name(taxonomy_list[0]),
```

Comment:
Do we know the files will be in every species folder?  Should we maybe use the source taxonomy?

Reply:
Good point. I don't think it would matter though. The genome would be downloaded again if it didn't exist for that species.